docs/usage.md

```diff
@@ -0,0 +1,145 @@
+# Usage
+
+## List Signup
+
+The most basic usage would be to direct users to a page where they can provide their email address to be notified of updates:
+
+Add a TemplateView for a landing page to your `urls.py`:
+
+    # project/urls.py
+    from django.views.generic import TemplateView
+
+    urlpatterns = [
+        [...]
+        url(r"^$", TemplateView.as_view(template_name="waitinglist/list_signup.html"), name="home"),
+    ]
+
+Update your template directories in `settings.py`:
```

Comment:
It would be necessary when using as an installed app in an existing project I believe. The usage section was created with this in mind rather than a Pinax starter project as I understand they work "out of the box" so to speak.